### PR TITLE
feat(Tabs): Adding a section for IgxBottomNav and IgxTabs changes in Update Guide.

### LINKED ISSUE #2696: Add BottomNav and Tabs sections to the Update Guide topic
Add sections for the IgxBottomNavComponent and IgxTabsComponent describing the changes we've made in 12.0.

### PR DESCRIPTION
**Pull request by gedinakova**
Closes #2696 

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/general/update-guide.md [hunk 1]
```diff
@@ -108,55 +108,106 @@ For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from th
...
+* The **API** of the components was changed during the refactoring.
+    * The `id`, `groups`, `viewTabs`, `contentTabs` and `tabs` properties were removed. Currently, the `items` property returns the collection of tabs.
+    * The following properties were changed:
+        * The tab item's `isSelected` was renamed to `selected`.
+    * The `tabItemSelected` and `tabItemDeselected` events were removed. We introduced three new events, `selectedIndexChanging`, `selectedIndexChange` and `selectedItemChange`, which provide more flexibility and control over the tabs' selection. Unfortunately, having an adequate migration for these event changes is complicated to say the least, so any errors should be handled at project level.
+
+    * **IgxTabs** specific members
+        * The `selectedTabItem` property was shortten to `selectedItem`.
+        * The `type` property, with its contentFit and fixed options, is no longer available. The header sizing & positioning mode is currently controlled by the `tabAlignment` input property which accepts four different values - start (default), center, end and justify. The old `fixed` type corresponds to the current `justify` alignment value.
```
Comment (astaev) [inline en/components/general/update-guide.md:204]: We can mention that old contentFit now corresponds to current start(not matter that it's the default one).

File: en/components/general/update-guide.md [hunk 2]
```diff
@@ -108,55 +108,106 @@ For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from th
...
- 
+
+### `IgxBottomNav` and `IgxTabs`
+The two components were completely refactored in order to provide more flexible and descriptive way to define tab headers and contents. It is recommended that you update via **ng update** in order to migrate the existing **igx-tabs** and **igx-bottom-nav** definitions to the new ones.
+
+* Template
+    * The new structure defines tab items each wrapping a header and a content. The headers may contain a label and/or an icon as well as any other custom content.
+    * For header styling purposes we introduced four new directives, namely `IgxBottomNavHeaderLabel`, `IgxBottomNavHeaderIcon`, `IgxTabHeaderLabel` and `IgxTabHeaderIcon`.
+    * Since the header component now allows adding any content, the `IgxTab` directive, which was previously used to retemplate the tab's header, was removed because it is no longer necessary.
+
```
Comment (astaev) [inline en/components/general/update-guide.md:167]: Just mention about routerLink directive - it's now set on the header but migrations should update it.

File: en/components/general/update-guide.md [hunk 3]
```diff
@@ -108,55 +108,106 @@ For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from th
...
+            </igx-tab-content>
+        </igx-tab-item>
+    ...
+    </igx-tabs>
+    ```
+* The **API** of the components was changed during the refactoring.
+    * The `id`, `groups`, `viewTabs`, `contentTabs` and `tabs` properties were removed. Currently, the `items` property returns the collection of tabs.
+    * The following properties were changed:
+        * The tab item's `isSelected` was renamed to `selected`.
+    * The `tabItemSelected` and `tabItemDeselected` events were removed. We introduced three new events, `selectedIndexChanging`, `selectedIndexChange` and `selectedItemChange`, which provide more flexibility and control over the tabs' selection. Unfortunately, having an adequate migration for these event changes is complicated to say the least, so any errors should be handled at project level.
```
Comment (DiyanDimitrov) [inline en/components/general/update-guide.md:200]: IMO We should split those for the two components. I am not sure those were called the same for the tabs and bottom nav components. For example the events of the bottom nav were called `onTabSelected`/`onTabDeselected`.

**Review by DiyanDimitrov — CHANGES_REQUESTED** [review]

File: en/components/general/update-guide.md [hunk 4]
```diff
@@ -108,55 +108,106 @@ For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from th
...
 
     | **Theme** | **Font Family** | **Type Scale** |
     |----------------|-----------------|-----------------|
     | Material | $material-typeface | $material-type-scale |
     | Fluent | $fluent-typeface | $fluent-type-scale |
     | Bootstrap | $bootstrap-typeface | $bootstrap-type-scale |
     | Indigo | $indigo-typeface | $indigo-type-scale |
- 
+
+### `IgxBottomNav` and `IgxTabs`
```
[the remark this answers is not in the document]
Comment (DiyanDimitrov) [reply en/components/general/update-guide.md]: I don't like them bold either 😄 It looks strange to have bold text in the header which already looks bold.

**Review by astaev — APPROVED** [review]

**Review by DiyanDimitrov — APPROVED** [review]

**Pull request merged**